### ISSUE TITLE
[False positive] `MD028` is raised when two note blocks are separated with one blank line

### ISSUE THREAD

**Issue by vepain** [DavidAnson/markdownlint#1610]
## Description of the false positive

`markdownlint` raises a `MD028` rule error when two note blocks are separated with one blank line.
However, the blanck line is required to process separatly the two blocks (otherwise the second one is not process and is considered as a text in the first block.

## MWE

```md
# Title

> [!WARNING]
> A warning

> [!NOTE]
> A note

```

## Behaviour solution proposed

* Do not raise this rule error
* Add a new lint rule to force one line separation between two note blocks

**Comment by DavidAnson** [comment]
Duplicate of: https://github.com/DavidAnson/markdownlint/issues/1419

**Comment by vepain** [comment]
I am so sorry, I haven't got the correct keywords when I was looking to an already existing issue...
I close this one, thank you.
